album.html

```diff
@@ -1,80 +1,76 @@
-                         <div class="seek-bar">
-                             <div class="fill"></div>
-                             <div class="thumb"></div>
-                         </div>
-                         <div class="current-time">2:30</div>
-                         <div class="total-time">4:45</div>
-                     </div>
-                     <h3 class="artist-name">Fallout Boy</h3>
-                 </div>
-                 <div class="player-bar-control-group right-controls">
-                     <div class="volume">
-                         <span class="ion-volume-high"></span>
-                         <div class="seek-bar">
-                             <div class="fill"></div>
-                             <div class="thumb"></div>
-                         </div>
-                     </div>
-
-                 </div>
-             </div>
-         </section>
-         <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
-         <script src="scripts/album.js"></script> 
-     </body>
- </html>
\ No newline at end of file
+<html>
+    <head>
+        <title>Bloc Jams</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Open+Sans:400,800,600,700,300">
+        <link rel="stylesheet" type="text/css" href="http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
+        <link rel="stylesheet" type="text/css" href="styles/normalize.css">
+        <link rel="stylesheet" type="text/css" href="styles/main.css">
+        <link rel="stylesheet" type="text/css" href="styles/album.css">
+        <link rel="stylesheet" type="text/css" href="styles/player_bar.css">
+    </head>
+    <body class="album">
+        <nav class="navbar">
+            <a href="index.html" class="logo">
+                <img src="assets/images/bloc_jams_logo.png" alt="bloc jams logo">
+            </a>
+            <div class="links-container">
+                <a href="collection.html" class="navbar-link">Collection</a>
+            </div>
+        </nav>
+        
+        <main class="album-view container narrow">
+            <section class="clearfix">
+                <div class="column half">
+                    <img src="assets/images/album_covers/01.png" class="album-cover-art">
+                </div>
+                <div class="album-view-details column half">
+                    <h2 class="album-view-title">The Colors</h2>
+                    <h3 class="album-view-artist">Pablo Picasso</h3>
+                    <h5 class="album-view-release-info">1909 Spanish Records</h5>
+                </div>
+            </section>
+            <table class="album-view-song-list">
+            </table>
+        </main>
+        
+        <section class="player-bar">
+            <div class="container">
+                <div class="control-group main-controls">
```

Comment:
We've changes some class names here. `player-bar-control-group` is now just `control-group` and `left-controls` and `right-controls` have been replaces with `main-controls` and `volume`, respectively.